mautic/mautic
# Report new columns

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | 
| New feature/enhancement? (use the a.x branch)      | 🟢 
| Deprecations?                          | 
| BC breaks? (use the c.x branch)        | 
| Automated tests included?              | Not relevant
| Related user documentation PR URL      | 
| Related developer documentation PR URL | 
| Issue(s) addressed                     | 



#### Description:

Added new columns to report section include Data Source to know what kind of report it is

![image](https://github.com/mautic/mautic/assets/462477/68407c56-dbd9-4fef-bbe5-08929849007c)




#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2.  Go to reports and check  it's OK
